### PR TITLE
GH-93: Resolve Placeholder in @TopicPartition

### LINKED ISSUE #93: @TopicPartition#partitions doesn't support property place holders and SpEL expressions
Example

```
@Component
public class Listener {
 
    @KafkaListener(topicPartitions = {
            @TopicPartition(topic = "${kafka.topic}", partitions = {"${kafka.topic-partition}"})
    })
    public void handle(Object msg, Acknowledgment ack) { }
}
```

Exception
```
Caused by: java.lang.NumberFormatException: For input string: "${kafka.topic-partition}"
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.lang.Integer.parseInt(Integer.java:481)
	at java.lang.Integer.valueOf(Integer.java:582)
	at org.springframework.kafka.annotation.KafkaListenerAnnotationBeanPostProcessor.resolvePartitionAsInteger(KafkaListenerAnnotationBeanPostProcessor.java:496)
	at org.springframework.kafka.annotation.KafkaListenerAnnotationBeanPostProcessor.resolveTopicPartitionsList(KafkaListenerAnnotationBeanPostProcessor.java:458)
	at org.springframework.kafka.annotation.KafkaListenerAnnotationBeanPostProcessor.resolveTopicPartitions(KafkaListenerAnnotationBeanPostProcessor.java:410)
	at org.springframework.kafka.annotation.KafkaListenerAnnotationBeanPostProcessor.processListener(KafkaListenerAnnotationBeanPostProcessor.java:368)
	at org.springframework.kafka.annotation.KafkaListenerAnnotationBeanPostProcessor.processKafkaListener(KafkaListenerAnnotationBeanPostProcessor.java:328)
	at org.springframework.kafka.annotation.KafkaListenerAnnotationBeanPostProcessor.postProcessAfterInitialization(KafkaListenerAnnotationBeanPostProcessor.java:262)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyBeanPostProcessorsAfterInitialization(AbstractAutowireCapableBeanFactory.java:422)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1583)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:545)
	... 39 more
```

### PR DESCRIPTION
**Pull request by garyrussell**
Fixes #93

### CODE REVIEW

File: spring-kafka/src/test/java/org/springframework/kafka/annotation/EnableKafkaIntegrationTests.java [hunk 1]
```diff
@@ -190,6 +192,18 @@ public void testJson() throws Exception {
 	public static class Config {
 
 		@Bean
+		public static PropertyPlaceholderConfigurer ppc() {
```
Comment (artembilan) [inline spring-kafka/src/test/java/org/springframework/kafka/annotation/EnableKafkaIntegrationTests.java]: That must be `PropertySourcesPlaceholderConfigurer` and you don't need to specify properties for it.
You can just rely on the default for PP:
```
partitions = "${zero:0}"
```

Comment (artembilan) [inline spring-kafka/src/test/java/org/springframework/kafka/annotation/EnableKafkaIntegrationTests.java]: But yeah, that's minor and is just an info for the future to simplify your codding life :smile: 

I can polishing it that way on merge.
Just let me know

Comment (garyrussell) [inline spring-kafka/src/test/java/org/springframework/kafka/annotation/EnableKafkaIntegrationTests.java]: ok; if you prefer - I don't mind either way.

**Comment by artembilan** [comment]
Merged as https://github.com/spring-projects/spring-kafka/commit/be016f0f7d1e74f3e0904ee35fd65f01fa877c86

**Pull request closed without merging**